src/Driver/PgSQL/Driver.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Doctrine\DBAL\Driver\PgSQL;
+
+use Doctrine\DBAL\Driver\AbstractPostgreSQLDriver;
+use ErrorException;
+use SensitiveParameter;
+
+use function addslashes;
+use function array_filter;
+use function array_keys;
+use function array_map;
+use function array_slice;
+use function array_values;
+use function assert;
+use function func_get_args;
+use function implode;
+use function pg_connect;
+use function pg_escape_string;
+use function pg_exec;
+use function pg_result_error;
+use function restore_error_handler;
+use function set_error_handler;
+use function sprintf;
+
+use const PGSQL_CONNECT_FORCE_NEW;
+
+final class Driver extends AbstractPostgreSQLDriver
+{
+    /** @inheritdoc */
```

Comment:
I don't recall the reason but this is usually done like this:

```suggestion
    /** {@inheritDoc} */
```

There are more occurrences in this PR.

Reply:
Looks like I did this wrong on the SQLite3 driver as well.